### DISCUSSION TITLE
Enter the Gungeon

### DISCUSSION THREAD

**Discussion by ghost** [skyline-emu/skyline#787] [Game Requests]
This game is super fun and easier to run compared to BOTW. Maybe one of the good games to start with.

**Comment by RafaelFF6844** [comment]
I'm with you, I wanna play Enter the Gurgaon on my mobile 💖

**Reply by AyderProgramic** [reply]
![Screenshot_2022-04-09-15-46-57-945_skyline emu](https://user-images.githubusercontent.com/74705634/162590425-4c1b051b-eee7-470d-9c63-66e5b71b6626.jpg)

**Reply by ghost** [reply]
Getting close!
